content/about-us/_index.md

```diff
@@ -15,7 +15,7 @@ aliases:
 
 ## Introduction
 
-The Carpentries is a fiscally sponsored project of [Community Initiatives](https://communityinitiatives.org/), a registered 501(c)(3) non-profit based in California, USA. The Carpentries teaches foundational coding and data science skills to researchers worldwide. We exist because the skills needed to do computational, data-intensive research are often not included as a part of basic research training in many disciplines. Our global community works together to provide:
+The Carpentries is a registered 501(c)(3) non-profit based in California, USA. The Carpentries teaches foundational coding and data science skills to researchers worldwide. We exist because the skills needed to do computational, data-intensive research are often not included as a part of basic research training in many disciplines. Our global community works together to provide:
```

Comment:
```suggestion
The Carpentries is a registered 501(c)(3) non-profit based in Delaware, USA. The Carpentries teaches foundational coding and data science skills to researchers worldwide. We exist because the skills needed to do computational, data-intensive research are often not included as a part of basic research training in many disciplines. Our global community works together to provide:
```

Reply:
based in Delaware, not CA